### PR TITLE
Trento-change#2: Revamp of HANA cluster discovery

### PR DESCRIPTION
**Pull request by tomschr**
### PR creator: Description

To include HANA scale-out clusters, diskless SBD clusters and maintenance status

### PR creator: Are there any relevant issues/feature requests?

* change#2
* https://confluence.suse.com/x/_YA0WQ

### CODE REVIEW

**Review by tomschr — COMMENTED** [review]
Thanks @abravosuse! I have found some smaller issues, nothing really serious. :slightly_smiling_face:

File: trento/xml/article_sap_trento.xml [hunk 1]
```diff
@@ -119,51 +119,96 @@ As agreed on https://confluence.suse.com/x/DAEcN on our Trento doc kick off
...
             Revamp of HANA cluster discovery to include HANA scale-out clusters,
             diskless SBD clusters and maintenance status.
           </remark>
-          Automated discovery of &sap; Systems, &sap; Instances,
-          &hana; clusters and ASCS/ERS clusters</para>
+          Automated discovery of:
+         - &sap; Systems and application instaces.
+         - multi-tenant &hana; databases and their instances.
+         - &hana; scale up clusters. 
+         - &hana scale out clusters.
```
[the remark this answers is not in the document]
Comment (abravosuse) [reply trento/xml/article_sap_trento.xml]: Yes, this should become a sublist of the existing list. But I wasn't sure about the syntax. So I left if for you to adapt it.

[the remark this answers is not in the document]
Comment (tomschr) [reply trento/xml/article_sap_trento.xml]: Ok, no problem. Let me know once you are finished.

[the remark this answers is not in the document]
Comment (abravosuse) [reply trento/xml/article_sap_trento.xml]: I am finished with this PR for the time being.

[the remark this answers is not in the document]
Comment (tomschr) [reply trento/xml/article_sap_trento.xml]: Ok, this is done now in commit 523902d

File: trento/xml/article_sap_trento.xml [hunk 2]
```diff
@@ -1283,61 +1346,67 @@ The health boxes in the dashboard are clickable. By clicking on one particular b
                 </para>
               </listitem>
               <listitem>
-                <para><guimenu>Stopped resources</guimenu> section: with a summary
-                of resources which have been stopped on the cluster.
+                <para><guimenu>&pace; Site details</guimenu> section: this ssection is split in three
```
Comment (tomschr) [inline trento/xml/article_sap_trento.xml:1349]: Typo:

```suggestion
                <para><guimenu>&pace; Site details</guimenu> section: this section is split in three
```

**Review by tomschr — COMMENTED** [review]
Thanks Alberto, I have only some small typos.

**Pull request merged**